book/src/specs/kimchi.md

```diff
@@ -188,9 +188,28 @@ fifth round:
 
 ## Constraint System Creation
 
+1. +3 on gates.len() here to ensure that we have room for the zero-knowledge entries of the permutation polynomial
+   see https://minaprotocol.com/blog/a-more-efficient-approach-to-zero-knowledge-for-plonk
+2. pad the rows: add zero gates to reach the domain size
 
 
-## Prover and Verifier Index Creation
+## Prover Index
```

Comment:
If I understand correctly, this index is dependent on the circuit itself? Like, this prover index is made effective only after the "general" SRS has been created? ( I refer to this question in the other PR as well, making a distinction between Setup and Index in papers)

Reply:
> If I understand correctly, this index is dependent on the circuit itself?

Yeah you basically compile the circuit to these two indexes